### PR TITLE
Fix edit view tabbing

### LINKED ISSUE #109: Tabbing is broken in the edit view
[Issuehunt badges](https://issuehunt.io/r/sindresorhus/gifski-app/issues/109)

The Dimensions dropdown has the first focus, if you tab you get Width, then Height, then the trimmer view. Instead of the trimmer view, it should have been the "pixels/percentage" dropdown.

When the edit view is presented, if you press Shift+Tab, it moves to focus to the trimmer view, if you press Esc then, the trimmer is hidden. We should find a way to prevent the trimmer from disappearing.

I was also thinking, maybe we should make the trimmer view the initial first responder instead of the "Dimensions" dropdown. You would then be able to press Space to view the video, which is nicer, I think. We could make it so that when you tab once, the dimension dropdown is focused.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $80.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($80.00)
### Submitted pull Requests
- [#132 Tabbing is broken in the edit view #109](https://issuehunt.io/r/sindresorhus/gifski-app/pull/132)
- [#141 Fix Edit window tabbing](https://issuehunt.io/r/sindresorhus/gifski-app/pull/141)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/sindresorhus/gifski-app/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by SergeyKuryanov**
Fixes #109 

Expectations:

- Edit view tab order should be correct.
- TrimmingView should not hide when user press ESC button (when has focus)
- TrimmingView should be initial responder so user can use space to play video after view appear.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#109: Tabbing is broken in the edit view](https://issuehunt.io/repos/119822304/issues/109)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

**Comment by sindresorhus** [comment]
> We could make it so that when you tab once, the dimension dropdown is focused.

Currently, if I press Tab after the edit view loads, the left trim handle is focused, instead it should tab right to the dimensions dropdown, ignoring the trim/progress handles.

File: Gifski/TrimmingAVPlayerViewController.swift [hunk 1]
```diff
@@ -131,4 +131,15 @@ final class TrimmingAVPlayerView: AVPlayerView {
...
+	// MARK: - Keyboard handling
+	let escKeyCode: UInt16 = 53
+
+	override func keyDown(with event: NSEvent) {
+		guard event.keyCode != escKeyCode else {
+			return
+		}
+
+		super.keyDown(with: event)
+	}
```
[the remark this answers is not in the document]
Comment (SergeyKuryanov) [reply Gifski/TrimmingAVPlayerViewController.swift]: No, I didn't. will try.

**Comment by sindresorhus** [comment]
@SergeyKuryanov Friendly bump :)

**Comment by SergeyKuryanov** [comment]
@sindresorhus delegate method added. But I didn't find a way to omit trimming handles. Maybe you have some suggestions?

**Comment by sindresorhus** [comment]
You could maybe use view traversing to find those handle views, like done in https://github.com/sindresorhus/Gifski/blob/master/Gifski/TrimmingAVPlayerViewController.swift and then set `. refusesFirstResponder = true` on them.

**Comment by sindresorhus** [comment]
Nice! Just one tiny thing left. The new "Loop Forever" checkbox is skipped when tabbing.

**Comment by SergeyKuryanov** [comment]
@sindresorhus fixed.

**Comment by sindresorhus** [comment]
Nice work! :)

**Pull request merged**